public/contextMenu.js

```diff
@@ -0,0 +1,409 @@
+/*
+MIT License
+
+Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+Original from: https://github.com/sindresorhus/electron-context-menu
```

Comment:
Maybe we should make it explicit (here or in some other place) the version of the original on which ours is based.

This will make it easier if we need to update the code with newer versions

Reply:
Added a reference to the specific release we based our code on on the url itself.